versions/src/main/java/cc/polyfrost/oneconfig/internal/renderer/LwjglManagerImpl.java

```diff
@@ -357,7 +357,7 @@ private static synchronized URL getJarFile() {
         ;
         //#if MC>11202 && MC<=11900
         //$$ String osArch = System.getProperty("os.arch");
-        //$$ jarName = jarName + "-" + ((osArch.startsWith("arm") || osArch.startsWith("aarch64")) ? "arm" : "noarm");
+        //$$ jarName = jarName + "-" + ((osArch.startsWith("arm") || osArch.startsWith("aarch64") || !osArch.contains("64")) ? "arm" : "noarm");
```

Comment:
?

Reply:
for 32-bit systems - its handled the same as ARM and in the same jar